environment-setup/ansible/TP2/group_vars/students.yml

```diff
@@ -0,0 +1,123 @@
+students:
+  - username: tstudent
+    pubkey: ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIGLr/eF/LPQ9kmX3LrpRfQhmlFg+UOZ0sgIvVsg+oH15 cunha@timbersaw
+  - username: alexandrearigoni
```

Comment:
Let's remove all entries in this file other than the `tstudent` one. We need to regenerate this file every term.